lucille-core/src/test/java/com/kmwllc/lucille/stage/CopyFieldsTest.java

```diff
@@ -62,11 +66,52 @@ public void testCopyFieldsSkip() throws Exception {
     assertEquals("input3 should be skipped.", doc.getStringList("output3").get(0));
   }
 
+  @Test
+  public void testCopyFieldsNested() throws Exception {
```

Comment:
suggest adding a test where we try to copy a dotted field to another dotted field with isNested=false and confirm that the field names are treated as literals
.e.g we have a field named "a.b.c" and we try to copy its value into "x.y.z"